### DISCUSSION TITLE
NTP always 1970

### DISCUSSION THREAD

**Discussion by jokerigno** [arendst/Tasmota#17731] [Q&A]
Hi all,

I'm struggling with a device that always has 1970 as time setting. I found an old issue https://github.com/arendst/Tasmota/issues/4243#issuecomment-491451743 referring to my same device but is quite old there's no direct solution so I was wondering if someone can suggest me hot to set it correctly? 

I already tried to set ntp server(s) 1 2 3 and reboot, also changed DNS adress and set static IP but nothing worked. Any other hint?

Thank you in advance

**Comment by barbudor** [comment]
Please set Weblog to 4 and provide logs from boot for 2 minutes
Also provide the output of command Status 5
Thanks

**Reply by sfromis** [reply]
You should also ensure that your network setup allows the device to actually communicate with the NTP servers you configure. Falling back to 1970 usually means failure of that.
